.circleci/config.yml

```diff
@@ -56,6 +55,101 @@ jobs:
 
 
 
+  build-linux-release:
+    <<: *defaults
+    docker:
+      - image: hyperledger/iroha-docker-develop
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - build-cache-release-{{ arch }}-{{ .Branch }}-
+            - build-cache-release-{{ arch }}-
+          paths:
+            - ~/.ccache
+      - run:
+          name: cmake
+          command: >
+            cmake \
+              -DCOVERAGE=OFF \
+              -DTESTING=OFF \
+              -H$IROHA_HOME \
+              -B$IROHA_BUILD \
+              -DCMAKE_BUILD_TYPE=Release \
+              -DPACKAGE_DEB=ON \
+              -DPACKAGE_TGZ=ON
+      - run:
+          name: make and package
+          command: |
+            cmake --build $IROHA_BUILD --target package -- -j4
+      - save_cache:
+          key: build-cache-release-{{ arch }}-{{ .Branch }}-{{ epoch }}
+          paths:
+            - ~/.ccache
+      - run:
+          name: copy deb file
+          command: |
+            cd $IROHA_BUILD
+            cp iroha-*.deb ./iroha.deb
```

Comment:
How about using specific filename instead of wildcard?

Reply:
it is not possible, because name looks like 
`iroha-0xd4cce615d9aaa8b96dd938b5ab67062f3ccfab24-Linux.deb`

Reply:
Use first 7 symbols, just like github?